pom.xml

```diff
@@ -201,7 +201,7 @@
 		<dependency>
 			<groupId>org.kohsuke</groupId>
 			<artifactId>github-api</artifactId>
-			<version>1.95</version>
+			<version>1.317</version>
```

Comment:
1.95 was using the hard-coded string URL for uploading the artifacts (upload.github.com), despite of the endpoint customization via `GitHub` builder. So I simply upgraded to the most recent version.